bmeg_app/views/literature_support_view.py

```diff
@@ -37,8 +34,7 @@
                     [
                         html.Label(i18n.t('app.widget_lit.menu1')),
                         dcc.Dropdown(id='gene_dd',
-                            options=[{'label': l, 'value': gid} for l,gid in lsu.gene_dd_selections(base_df,'geneID','gene').items()],
-                            value=[{'label': l, 'value': gid} for l,gid in lsu.gene_dd_selections(base_df,'geneID','gene').items()][0]['value'],
+                            value="MTOR/ENSG00000198793", search_value="MTOR/ENSG00000198793"
```

Comment:
add value to config file?

Reply:
Yes, but let's do that in a later PR that addresses default values used by all the different views